Source/SpeechToTextV1/SpeechToText.swift

```diff
@@ -850,6 +887,9 @@ public class SpeechToText {
         speakerLabels: Bool? = nil,
         grammarName: String? = nil,
         redaction: Bool? = nil,
+        processingMetrics: Bool? = nil,
+        processingMetricsInterval: Double? = nil,
+        audioMetrics: Bool? = nil,
```

Comment:
Do these need to move to the end of the parameters?

Reply:
In Swift, we can add new optional parameters anywhere in the method signature as long as they have default values, since any invocations that don't have the parameters (initially _all_ of them) will simply get the default value.

Once the parameters have been added, after that they can't be moved without the risk of breaking some users.